gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5.1-all.zip
```

Comment:
@danimontoya this version is for Arrow Meta. It's not involved in this case.

There are 2 versions that should be the same:
* That version in that file in `arrow-typeproofs` repository
* `BASE_GRADLE_VERSION` in this repository which is used to run the test in `arrow-typeproofs` repository. that's the reason why they must be the same version.